### ISSUE TITLE
Dropdown Component has WCAG2 Violations

### ISSUE THREAD

**Issue by mbquinonez** [primefaces/primeng#9760]
**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
Dropdown component has misplaced role attribute which causes failure of wcag2 test https://dequeuniversity.com/rules/axe/4.1/aria-command-name

**Expected behavior**
Dropdown should should pass wcag2 tests

**Minimal reproduction of the problem with instructions**
Use Dropdown component and run accessibility tools such as axe

**What is the motivation / use case for changing the behavior?**
To pass automated accessibility audits

**Please tell us about your environment:**

Angular version: 11

PrimeNG version: 11

Browser: Chrome

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
